src/pages/receive/receive.scss

```diff
@@ -36,36 +32,27 @@ page-receive {
     }
   }
   .qr-container {
-    height: 100%;
-    display: flex;
-    flex-direction: column;
-    align-items: center;
-    justify-content: center;
-
     text-align: center;
-    .icon {
-      vertical-align: middle;
-      margin: 0 10px;
-    }
     .text-address {
-      margin: 5px 0 0 0;
-      font-size: 12px;
+      font-size: 10px;
+      margin-bottom: 20px;
       color: color($colors, dark);
+      @media (min-height: 525px) {
+        font-size: 12px;
+      }
       > img {
         vertical-align: middle;
         margin-right: 5px;
         width: 16px;
         display: inline-block;
       }
     }
-    img {
-      width: auto;
-      display: inherit;
+    .aclass {
+      display: flex;
```

Comment:
We now have a mixin called `center` for these 3 lines in `mixins.scss`. It can be used via `@include center;`

Reply:
fixed up